app/js/app.js

```diff
@@ -35,6 +35,57 @@ window.addEventListener('DOMContentLoaded', () => {
     console.error('[Customizer] An error occurred getting the manifestURL');
   };
 
+  /**
+   * XXX - TEMP: Bug 1148139 - [Customizer] Inline stylesheets in JavaScript
```

Comment:
We should file a follow-up bug for removing this, rather than adding the bug number where it was added. You can already gather that information easily using ```git blame```.